docs/website/docs/plus/intro.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Introduction
+description: Introduction to dlt+
+---
+
+import Link from '../_plus_admonition.md';
+
+<Link/>
```

Comment:
I'd make an exception for this particular page and not include the admonition to it: I think it introduces visual clutter without helping much: this page already elaborates on what's dlt+ is and has the caution admonition linking the waiting list/licence.

Reply:
I think it's important to let people know from the very beginning how they can distinguish the dlt+ pages